cmd/logcli/query.go

```diff
@@ -83,7 +84,14 @@ func doQuery() {
 			labels = padLabel(ls, maxLabelsLen)
 		}
 
-		printLogEntry(i.Entry().Timestamp, labels, i.Entry().Line)
+		switch *outputMode {
```

Comment:
Do you mind moving this whole switch block to a function in utils.go which can also be used in tail.go?

Reply:
This is not straight forward, because of https://github.com/grafana/loki/pull/731#discussion_r303296330. `default` mode requires only a reduced amount of labels, while `jsonl` uses all labels. This means there is no common interface between those. 

Furthermore, it seems hard to move the common label code into `printLogEntry`, because it is called one at a time.

Reply:
Maybe you can pass on the raw labels and let each function have separate handling for it.
It would be easier to maintain the code since the same switch block is used in tail.go as well.

Reply:
But we would not be able to know which labels are common, because we only see one line at a time?

Reply:
Yeah, you are right. Sorry I missed that.

Reply:
Nevertheless, we could pass common labels as a parameter, which would keep the current functionaltiy

Reply:
Yes, that should also work.

Reply:
Okay, refactored this. Looks clean now :)